apps/tx-builder/src/pages/Dashboard.tsx

```diff
@@ -62,17 +36,76 @@ const Dashboard = (): ReactElement => {
     setContract(interfaceRepo.getMethods(abi))
   }, [abi, interfaceRepo])
 
-  const isAddressInputFieldValid = isValidAddress(address) || !address
+  const isAbiAddressInputFieldValid = !abiAddress || isValidAddress(abiAddress)
 
-  const contractHasMethods = contract && contract.methods.length > 0 && !isABILoading
+  const contractHasMethods =
+    abiStatus === FETCH_STATUS.SUCCESS && contract && contract.methods.length > 0
 
-  const isTransferTransaction = isValidAddress(address) && !abi && !isABILoading
-  const isContractInteractionTransaction = abi && contract && !isABILoading
+  const isTransferTransaction =
```

Comment:
I think that there is an issue with the `isTransferTransaction` condition.

If I use a EOA address the form is not showed:

<img width="1221" alt="Captura de pantalla 2022-07-18 a las 14 35 28" src="https://user-images.githubusercontent.com/26763673/179512448-b24df162-2e60-4edf-bb58-9ebdcc055fed.png">


![Captura de pantalla 2022-07-18 a las 14 26 50](https://user-images.githubusercontent.com/26763673/179511921-1d7a2113-0762-4cd6-bb19-a4bbf4f60cc0.png)

Reply:
Fixed senor